src/sass/main.scss

```diff
@@ -94,3 +94,8 @@
 
 @import 'overrides/overrides.scss';
 @import 'overrides/print.scss';
+
+// Required because Google Auto-complete needs to appear above modal: https://github.com/twbs/bootstrap/issues/4160
```

Comment:
@Anoninnyc @DaleMcGrew I'm going to move this since it conflicts with my commit already, but for future reference, please ALSO be specific in commenting what a hack or override you're introducing addresses. I was able to tell from the PR/commit in this case, but because `.pac-container` is not in the codebase, it is much harder to figure out if an override is still relevant. Just FYI!